Client/Frontend/Widgets/AlternateSimpleHighlightCell.swift

```diff
@@ -0,0 +1,199 @@
+            make.leading.equalTo(contentView).offset(AlternateSimpleHighlightCellUX.CellSideOffset)
+            make.height.equalTo(76)
+            make.width.equalTo(99)
+        }
+
+        selectedOverlay.snp_remakeConstraints { make in
+            make.edges.equalTo(contentView)
+        }
+
+        domainLabel.snp_remakeConstraints { make in
+            make.leading.equalTo(siteImageView.snp_trailing).offset(AlternateSimpleHighlightCellUX.CellTopBottomOffset)
+            make.top.equalTo(siteImageView).offset(-2)
+            make.bottom.equalTo(titleLabel.snp_top).offset(-4)
+        }
+
+        titleLabel.snp_remakeConstraints { make in
+            make.leading.equalTo(siteImageView.snp_trailing).offset(AlternateSimpleHighlightCellUX.CellTopBottomOffset)
+            make.trailing.equalTo(contentView).inset(AlternateSimpleHighlightCellUX.CellSideOffset)
+        }
+
+        descriptionLabel.snp_remakeConstraints { make in
+            make.leading.equalTo(statusIcon.snp_trailing).offset(AlternateSimpleHighlightCellUX.TitleLabelOffset)
+            make.bottom.equalTo(statusIcon)
+        }
+
+        timeStamp.snp_remakeConstraints { make in
+            make.trailing.equalTo(contentView).inset(AlternateSimpleHighlightCellUX.CellSideOffset)
+            make.bottom.equalTo(descriptionLabel)
+        }
+
+        statusIcon.snp_remakeConstraints { make in
+            make.size.equalTo(SimpleHighlightCellUX.StatusIconSize)
+            make.leading.equalTo(titleLabel)
+            make.bottom.equalTo(siteImageView).priorityLow()
+            make.top.greaterThanOrEqualTo(titleLabel.snp_bottom).offset(6).priorityHigh()
+            make.bottom.lessThanOrEqualTo(contentView).offset(-AlternateSimpleHighlightCellUX.CellTopBottomOffset).priorityHigh()
+        }
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setImageWithURL(url: NSURL) {
+        siteImageView.sd_setImageWithURL(url) { (img, err, type, url) -> Void in
+            guard let img = img else {
+                return
+            }
+            // Resize an Image to a specfic size to make sure that it doesnt appear bigger than it needs to (32px) inside a larger frame (48px).
+            self.siteImageView.image = img.createScaled(CGSize(width: 32, height: 32))
+            self.siteImageView.image?.getColors(CGSizeMake(25, 25)) { colors in
+                self.siteImageView.backgroundColor = colors.backgroundColor ?? UIColor.lightGrayColor()
+            }
+        }
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        self.siteImageView.image = nil
+        self.timeStamp.text = nil
+    }
+
+    func configureWithSite(site: Site) {
+        if let icon = site.icon, let url = NSURL(string:icon.url) {
+            self.setImageWithURL(url)
+        } else {
+            let url = NSURL(string: site.url)!
```

Comment:
Nit: Theres a nice helper you can use for this -> `site.url.asURL!` instead of `NSURL(...)`